crates/kitsune_p2p/kitsune_p2p/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## \[Unreleased\]
 
+- Gossip send failures and target expired events are now logged as warnings rather than errors, and have additional text for claity
```

Comment:
```suggestion
- Gossip send failures and target expired events are now logged as warnings rather than errors, and have additional text for clarity.
```